transhumandesign/kag-base
# Removed buggy grass from maps

## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

The maps "Princess01" (Save the Princess) and "EarthMoving" (TTH) had grass tiles placed above solid ground.
It looks odd and buggy. In my opinion, bugs in tutorial modes (Save the Princess) have potential to make a bad impression on new players.
So I made this change which removes those grass tiles. 
These are the only maps I could find with this problem.

https://i.imgur.com/NSPbWix.png

## Steps to Test or Reproduce

- Go to Sandbox
- Open console
- /loadmap "Princess01"
- /loadmap "EarthMoving"
- Notice the buggy grass tiles are gone.
